nltk/test/unit/test_concordance.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+
+import unittest
+from nose import with_setup
+
+from nltk.corpus import gutenberg
+from nltk.text import Text
+
+import contextlib
+import sys
+
+try:
+    from StringIO import StringIO
+except ImportError as e:
+    from io import StringIO
+
+
+@contextlib.contextmanager
+def stdout_redirect(where):
+    sys.stdout = where
+    try:
+        yield where
+    finally:
+        sys.stdout = sys.__stdout__
+
```

Comment:
Actually, this context manager can be avoided using the `file` parameter in Python3 style `print_function`. Use `from __future__ import print_function` and 

```
print(somestr, file=somewhere)  # prints to *somewhere*
print(somestr, file=sys.stdout)  # default setting prints to stdout.
```

See https://docs.python.org/3/library/functions.html#print